lessons/day_06/python/resolvers.py

```diff
@@ -0,0 +1,173 @@
+from ariadne import QueryType, ObjectType, EnumType, MutationType
+from ariadne import InterfaceType, UnionType
+from random import randint
+from models import Skill, Person
+from data import session
+from datetime import datetime
+import uuid
+
+
+def create_persons(info, input):
+    friends = []
+    skills = []
+    if 'friends' in input:
+        person_ids = input.pop('friends')
+        friends = session.query(Person).filter(Person.id.in_(person_ids)).all()
+    if 'skills' in input:
+        skill_ids = input.pop('skills')
+        skills = session.query(Skill).filter(Skill.id.in_(skill_ids)).all()
+
+    person = Person(**input)
```

Comment:
variable name defined in outer scopes https://github.com/thinkb4/a-walk-in-graphql/blob/77841ecd9e37614f36fd488ebf49e236de1afd6a/lessons/day_06/python/resolvers.py#L38